Logger.Tests/LogFactoryTests.cs

```diff
@@ -6,6 +6,31 @@ namespace Logger.Tests
     [TestClass]
     public class LogFactoryTests
     {
-        
+        [TestMethod]
+        public void CreateLogger_NullFilePath_ReturnsNullLogger()
+        {
+            string? filePath = null;
+
+
+            LogFactory loggerfactory = new LogFactory();
+            loggerfactory.ConfigureFileLogger(filePath);
+            // baseLogger.Log();
+            BaseLogger? baselogger = loggerfactory.CreateLogger();
```

Comment:
```suggestion
            BaseLogger? baselogger = loggerfactory.CreateLogger(nameof(LogFactoryTests));
```
Here is where you'd use `nameof` for your className